clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -1113,7 +1113,7 @@ public String documentation() {
     private static String toHtml() {
         DocumentedType[] types = {
             BOOLEAN, INT8, INT16, INT32, INT64,
-            UNSIGNED_INT32, VARINT, VARLONG, UUID, FLOAT64,
```

Comment:
Maybe we should document "all" types in the protocol guide even though some types are unused yet.

Reply:
This makes sense. I added `UINT16` and `COMPACT_RECORDS` without removing anything.
I skipped the tagged fields though, as referencing to them also needs fixing throughout the page and it's probably better be done separately.